sw/host/cryptotest/testvectors/parsers/wycheproof_ecdsa_parser.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import json
+import jsonschema
+import logging
+import sys
+
+from Crypto.Util.asn1 import DerSequence
+
+# Mapping from the curve names used by Wycheproof to those used by cryptotest
+EC_NAME_MAPPING = {
+    "secp256k1": "p256",
+    "secp384r1": "p384",
+}
+
+
+def _parse_der_signature(der_str):
+    seq_der = DerSequence()
+    seq_der.decode(
+        bytes.fromhex(der_str),
+        nr_elements=2,
+        only_ints_expected=True,
+        strict=True,
+    )
+    seq = seq_der[:]  # Convert the DerSequence object to a list
+
+    # Some tests parse correctly as a DER sequence but produce extra values.
+    # These tests should be reject for a bad DER-encoded signature
+    if len(seq) != 2:
+        raise ValueError("Failed to parse DER-encoded signature into r and s values")
+    return seq
+
+
+def parse_test_vectors(raw_data):
+    test_groups = raw_data["testGroups"]
+    test_vectors = list()
+    for group in test_groups:
+        # Parse the key for the group
+        key = group["key"]
+
+        # Parse tests within the group
+        for test in group["tests"]:
+            logging.debug(f"Parsing tcId {test['tcId']}")
+            test_vec = {
+                "algorithm": "ecdsa",
+                "operation": "verify",
+                "curve": EC_NAME_MAPPING[key["curve"]],
+                "hash_alg": group["sha"].lower(),
+                "message": list(bytes.fromhex(test["msg"])),
```

Comment:
Are the messages in the test vectors not given as large integers instead of hex? I only see digits 0-9 in the `msg` fields, not the characters a-f.